### PR TITLE
Add Aspire 2017: osu!catch showcase announcement

### PR DESCRIPTION
**Pull request by JBHyperion**
### Description

Add osu!catch Aspire showcase announcement for 27/01/18

### Status

Pending Review - mostly want clarification on the reference to "2017" Aspire since this is late coming out and it's now 2018...

### Changelog

23/01/18 - Initial PR

### CODE REVIEW

File: news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Aspire 2017: osu!catch Voting Opens Soon"
+date: 2018-01-24 09:00:00 +0000
+---
+
+After a long wait of preparation and scrutinisation by our panel of expert judges, **Aspire 2017: osu!catch** is finally ready to step into the spotlight! Join us this **Saturday the 27th of January at 23:00 UTC** for a livestream preview showcasing the final ten submissions, featuring expert commentary from our panel of talented judges; [Ascendance](https://osu.ppy.sh/users/2931883), [Nokashi](https://osu.ppy.sh/users/5431196), [Sartan](https://osu.ppy.sh/users/4100941) and [Zak](https://osu.ppy.sh/users/1375955)!
```
Comment (MegaApplePi) [inline news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md]: `Join us this **Saturday the 27th of January at 23:00 UTC**` ->
`Join us this **Saturday, the 27th of January, at 23:00 UTC**`
🤷‍♂️

File: news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![](/wiki/shared/news/banners/osu_catch.png)
+
+With some incredible offerings from both experienced osu!catch Aspirants and fresh-faced visionaries alike, the Aspire 2017: osu!catch showcase is sure to serve up a treat on all creative fronts. And if that wasn't enough, you'll be able to see all the entries playtested by our final judge, [alienflybot](https://osu.ppy.sh/users/636114)! 
+
+There will be eye-popping aesthetics and gameplay alike, from snazzy storyboards to astonishing pattern art, teleporting catchers to mountainous fruit stacks - if you want to know what osu!catch mapping is truly capable of, this is the showcase for you! Even if you're not an avid osu!catch player, fruits are good for you, so what's to lose by tuning in!?
+
+We'll be starting at **Saturday the 27th of January at 23:00 UTC** over on the [official osu! Twitch channel](https://twitch.tv/osulive). Voting will go live as soon as the stream begins, so you can vote along with everybody watching and select your most creative and astounding fruit-arrangement masterpieces!
```
Comment (MegaApplePi) [inline news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md]: rephrase the first sentence to: `We'll be starting the live stream over at the [official osu! Twitch channel](https://twitch.tv/osulive) on **Saturday the 27th of January at 23:00 UTC**.`

if you don't like that ☝️ , then at least change `at **Saturday` to `on **Saturday`

File: news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md (hunk 1 above)
Comment (JBHyperion) [reply news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md]: Pause in speech isn't really implied here so I didn't feel the need to add commas I guess?

File: news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md (hunk 2 above)
Comment (JBHyperion) [reply news/2018-01-23-aspire-2017-osucatch-voting-opens-soon.md]: Rearranged this sentence as you suggested for some swag variety yo

**Pull request merged**